test.py

```diff
@@ -1,32 +1,38 @@
-##############################################################################
-# Copyright 2019 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#   https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#############################################################################
+""""
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
 
 import pytest  # noqa: F401; pylint: disable=unused-variable
 from utils import generate_keys
 
 
-# Test to check if objects are being produced from the generate_keys()
-# function.
 def test_generate_keys_1():
```

Comment:
In general we try to make the names descriptive as it helps when it fails to quickly figure out why the change may have caused the test to fail. Something like `test_generate_keys_returns_bytes` or something of that sort?

I realize this is a bit out of scope of the PR so you can just keep this in mind for your next change if you would like.